### DISCUSSION TITLE
Stereo Error

### DISCUSSION THREAD

**Discussion by Funky-Synatra** [lucidrains/BS-RoFormer#32] [Q&A]
I'm getting this when trying to separate an audio file:

''''
AssertionError: stereo needs to be set to True if passing in audio signal that is stereo (channel dimension of 2). also need to be False if mono (channel dimension of 1)
'''

**Comment by lucidrains** [comment]
@Drausio72 you just have to set this flag to true https://github.com/lucidrains/BS-RoFormer/blob/main/bs_roformer/bs_roformer.py#L256

**Reply by Funky-Synatra** [reply]
Thanks! That was easy :)
Still couldn't make it work, but I'll get there :)

**Reply by lucidrains** [reply]
oh what's the new error

btw, someone has already trained a model and open sourced it

**Reply by Funky-Synatra** [reply]
I saw it today. It's available on UVR5, right?

**Reply by ghost** [reply]
Some BS/Mel-Roformers models and inference code are available on this repo: https://github.com/ZFTurbo/Music-Source-Separation-Training

The same models can be used in UVR but it's currently only in beta phase and it has some bugs (**don't use overlap > 6 with these models !**)
The beta can be downloaded here: https://github.com/TRvlvr/model_repo/releases/download/uvr_update_patches/UVR_Patch_4_14_24_18_7_BETA_full_Roformer.exe
And then you have to use UVR's download center (in options window) to get the models (they are listed under "MDX" category).

**Reply by Funky-Synatra** [reply]
Thanks! I'll take a look at the models.
